mitmproxy/tcp.py

```diff
@@ -47,5 +50,10 @@ def __init__(self, client_conn, server_conn, live=None):
         messages=List[TCPMessage]
     )
 
+    def copy(self):
```

Comment:
The default implementation should give a deep copy, so if it doesn't do that, it's a bug.